QNormalizModule.cpp

```diff
@@ -610,7 +603,7 @@ PyObject* _NmzCompute(Cone<Integer>* C, PyObject* args)
         }
     }else{
         to_compute = PyList_New( arg_len - 1 );
-        for( int i = 1;i<arg_len;i++){
+        for( int i = 1;i<arg_len;i++){  // Is i the correct index?? -mkoeppe
```

Comment:
Yes, I am indeed. `args[0]` is the cone.

Reply:
But you are using it also as an index into `to_compute`, which seems off by one.
Is this tested?

Reply:
In Sage, I don't actually use NmzCompute at the moment, so this bug can't be the cause.